tests/baselines/reference/jsxParsingError1.errors.txt

```diff
@@ -15,10 +14,8 @@ tests/cases/conformance/jsx/file.tsx(11,46): error TS1161: Unterminated regular
     const class1 = "foo";
     const class2 = "bar";
     const elem = <div className={class1, class2}/>;
-                                       ~
-!!! error TS1005: '}' expected.
-                                               ~
-!!! error TS1003: Identifier expected.
-                                                 
-!!! error TS1161: Unterminated regular expression literal.
+                                 ~~~~~~
+!!! error TS2695: Left side of comma operator is unused and has no side effects.
+                                 ~~~~~~~~~~~~~~
+!!! error TS18007: JSX expressions may not use the comma operator. Did you mean to write an array?

```

Comment:
These two errors make a lot more sense to me than the previous three, although I could do without the first. I considered making the error range start on the comma operator instead of at the beginning of the expression. Thoughts?